api/app.py

```diff
@@ -26,13 +27,11 @@
 calendar_service = GoogleCalendarService(client)
 event_service = GoogleEventsService(client)
 travel_service = GoogleTravelService(maps_client, env.google.apikey)
-rabbit_connection = create_connection(
-    env.rabbit.host_out,
-    env.rabbit.port_out,
-    env.rabbit.connection_attempts,
-    env.rabbit.retry_delay
-)
-rabbit_channel = create_rabbit_channel(rabbit_connection, env.rabbit.exchange_out, env.rabbit.topic_out)
+rabbit_channel = RabbitChannel.create(ChannelEnvironment(env.rabbit.exchange_out, env.rabbit.topic_out),
+                                      RabbitEnvironment(env.rabbit.host_out,
+                                                        int(env.rabbit.port_out),
+                                                        int(env.rabbit.connection_attempts),
+                                                        int(env.rabbit.retry_delay)))
```

Comment:
Can we cast those `int` params while reading the env?

Reply:
done